app/settings/fiware-glancesync-logging.cfg

```diff
@@ -0,0 +1,39 @@
+[loggers]
+keys=root,client
+
+[handlers]
+keys=fileHandler, streamHandler
+
+[formatters]
+keys=simpleFormatter, clientFormatter
+
+[logger_root]
+level=DEBUG
+handlers=fileHandler
+qualname=GlanceSync-API
+propagate=0
+
+[logger_client]
+level=DEBUG
+handlers=streamHandler
+qualname=GlanceSync-Client
+propagate=0
+
+[handler_fileHandler]
+class=logging.handlers.RotatingFileHandler
+level=DEBUG
+formatter=simpleFormatter
+filename: glancesync-api.log
```

Comment:
This path could be a big issue due to a future pip installation.
Log files should be written in /var/log/{component_name}/{file_name}.